docs/mmds/mmds-user-guide.md

```diff
@@ -175,12 +175,66 @@ Output:
 
 ### Retrieving metadata in the guest operating system
 
-To retrieve existing MMDS metadata from guest operating system, an HTTP `GET`
+Accessing the contents of the metadata store from guest operating system
+can be done using one of the following methods:
+- `V1`: simple request/response method
+- `V2`: session-oriented method
+
+#### Version 1
+
+To retrieve existing MMDS metadata using MMDS version 1, an HTTP `GET`
```

Comment:
I would add that MMDS v1 does not support PUT.

Reply:
Done